keps/sig-windows/2802-identify-windows-pods-apiserver-admission/kep.yaml

```diff
@@ -42,3 +42,5 @@ disable-supported: true
 
 # The following PRR answers are required at beta release
 metrics:
+  - kube_pod_status_phase
+  - apiserver_request_total
```

Comment:
Can you add some details to the metrics here? Are new labels being made for existing metrics for this enhancement? Can you add a short of how these metrics apply to this enhancement?
PRR review will probably ask these questions too.

Reply:
Both are existing metrics. I added [info on how to use them](https://github.com/kubernetes/enhancements/pull/3102/files#diff-0bd86b6b4704c4e10a453c7f15e74e31aacb53891087cf069ae11b8bf5293a30L314-L319) in the last KEP iteration itself. Let me know if you want it to expanded a bit more.

Reply:
Ah, thanks.
If there are details in the KEP that addresses my concerns.